### ISSUE TITLE
Client still can receive an action from channel that it was unsubscribed from but not received a confirmation about

### ISSUE THREAD

**Issue by mifopen** [logux/logux#117]
I.e. calling
`client.sync(logux/unsubscribe...)`
doesn't mean we won't process actions from this channel starting from the next tick.
Is it by design and do apps have to handle such cases by themselves?

We are deinitializing part of the app state at the same time we're unsubscribing from some channels and we were not ready to receive channel actions from what we assumed to be a closed topic.

**Comment by ai** [comment]
Yes, It is by design done it is work in the cases so apps that I saw.

What case do you have when extra action could break your app?

You can add a logic by adding custom input actions filter (check our encrypt actions file for an example).
